syntax = "proto3";

import "timeUiPb.proto";

message EntityUiPb {
    string id = 1;
    StatusEnum lifeTime = 2;
    LocaleUiPb locale = 3;
}

message LocaleUiPb{
    TimeZoneEnum defaultTimeZone = 1;
}

enum StatusEnum {
    UNKNOWN_STATUS = 0;
    ACTIVE = 1;
    DELETED = 2;
    SUSPUNDED = 3;
    BLOCKED = 4;
}
